### PR TITLE
Support ToUndirected

### PR DESCRIPTION
**Pull request by DarthMax**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
   * `link_features` (only for LP models)
   * `node_property_steps`
 * Added new convenience factory methods to create pipeline objects.
-
+* Added a new method `gds.graph.relationships.to_undirected` to turn a directed relationship type to an undirected, when using GDS >= 2.3.0
```
Comment (Mats-SX) [inline changelog.md:20]:
```suggestion
* Added a new endpoint `gds.beta.graph.relationships.to_undirected` to turn a directed relationship type to an undirected, when using GDS >= 2.3.0
```

**Review by Mats-SX — CHANGES_REQUESTED** [review]
The blocking comment is for the test.

**Review by Mats-SX — APPROVED** [review]

File: graphdatascience/graph/graph_entity_ops_runner.py [hunk 2]
```diff
@@ -181,6 +181,22 @@ def stream(self, G: Graph, relationship_types: List[str] = ["*"], **config: Any)
...
+    @compatible_with("toUndirected", min_inclusive=ServerVersion(2, 3, 0))
+    @graph_type_check
+    def to_undirected(
+        self, G: Graph, relationship_type: str, mutate_relationship_type: str, **config: Any
+    ) -> "Series[Any]":
+        self._namespace += ".toUndirected"
+        query = f"CALL {self._namespace}($graph_name, $config)"
+
+        actual_config = config | {
+            "relationshipType": relationship_type,
```
[the remark this answers is not in the document]
Comment (DarthMax) [reply graphdatascience/graph/graph_entity_ops_runner.py]: Yeah unfortunately it does only work with Pyton >= 3.9

[the remark this answers is not in the document]
Comment (Mats-SX) [reply graphdatascience/graph/graph_entity_ops_runner.py]: 😢

File: changelog.md (hunk 1 above)
Comment (Mats-SX) [inline changelog.md:20]:
```suggestion
* Added a new endpoint for the new procedure `gds.beta.graph.relationships.toUndirected` in GDS 2.3
```

**Pull request merged**